nmr_commons/tracking/Trajectory.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import csv
+import copy
+
+
+class Trajectory:
+    """Trajectory class where trajectory is a list of point and each point is a tuple of two points in space"""
+    ####
+    # Constructor
+    def __init__(self, items=None, norm_items=None):
+        # self.list = items if items else []    #this was before, I worried about mutable parameters
```

Comment:
Do not commit commented code, it tends to stay in this form forever and to have less and less in common with surrounding code.
Besides that, this version (inferring from class description) is perfectly fine as long as `items` is indeed list of tuples.